include/deal.II/lac/constraint_matrix.h

```diff
@@ -1089,6 +1089,18 @@ class ConstraintMatrix : public Subscriptor
    *
    * @ingroup Exceptions
    */
+  DeclException1 (ExcColumnNotStoredHere,
+                  size_type,
+                  << "The index set given to this constraint matrix indicates "
+                  << "constraints for degree of freedom " << arg1
+                  << " should not be stored by this object, but a constraint "
+                  << "uses this degree of freedom.");
```

Comment:
Yes, you are right. This is for a constraint that references a DoF that is not part of the index set.
I rephrased the error message.